manifests/config.pp

```diff
@@ -16,7 +16,7 @@
           ensure  => directory,
           owner   => 0,
           group   => 0,
-          mode    => '0664',
+          mode    => '0775',
```

Comment:
Thanks! It looks like there are some tests that actually verify that the resource mode is 664, so if you could update the tests that would be great!